packages/runtime/src/widgets/FittedColumn.tsx

```diff
@@ -0,0 +1,105 @@
+import { useCallback, useMemo } from "react";
+import { Col } from "antd";
+import { indexOf, keys } from "lodash-es";
+import {
+  CustomScopeProvider,
+  useCustomScope,
+  useRegisterBindings,
+  useTemplateData,
+} from "@ensembleui/react-framework";
+import type { CustomScope } from "@ensembleui/react-framework";
+import { WidgetRegistry } from "../registry";
+import { EnsembleRuntime } from "../runtime";
+import type { FlexboxProps } from "../shared/types";
+import { getColor, getCrossAxis, getMainAxis } from "../shared/styles";
+// eslint-disable-next-line import/no-cycle
+import { useEnsembleAction } from "../runtime/hooks/useEnsembleAction";
+
+export const FittedColumn: React.FC<FlexboxProps<true>> = (props) => {
```

Comment:
Very similar to `Flex`, reuse the Column widget and just extract out the special attributes for FittedColumn.